lib/opencensus/trace.rb

```diff
@@ -36,7 +36,7 @@ class << self
       #
       # @param [SpanContext] span_context
       #
-      def set_span_context span_context
+      def span_context= span_context
```

Comment:
If we're going to do this, then we probably should rename `current_span_context` to `span_context`.

Reply:
Sure, will do.